llvm/test/DebugInfo/ARM/align-func-start.ll

```diff
@@ -0,0 +1,27 @@
+; RUN: llc -mtriple=arm-none-eabi < %s | FileCheck %s
+; RUN: llc -mtriple=arm-none-eabi < %s | llvm-mc --triple=arm-none-eabi -mcpu=cortex-m3
```

Comment:
Do we need this 2nd run line?

Reply:
It checks for the original way we discovered this problem, which was that `llc` generated assembly which was rejected by the assembly parser.

Reply:
Ah I see, ok cool. Potentially worth adding a comment saying as such? ymmv, still LGTM either way

Reply:
Done, thanks.